models/topic.go

```diff
@@ -51,6 +54,11 @@ func (err ErrTopicNotExist) Error() string {
 	return fmt.Sprintf("topic is not exist [name: %s]", err.Name)
 }
 
+// TopicValidator checks topics by length and match pattern rules
+func TopicValidator(topic string) bool {
```

Comment:
I don‘t like the name of this function because it sounds like a struct/interface, Please rename it to `ValidateTopic`.